extensions/interactions/DragAndDropSort/directives/drag_and_drop_sort_interaction_directive.html

```diff
@@ -0,0 +1,63 @@
+<style>
+  .drag-and-drop-sort-form-container {
+    background: #fff;
+    max-width: 100%;
+    padding: 8px;
+  }
+  @media screen and (max-width: 959px) {
+    .drag-and-drop-sort-form-container {
+      width: 100vw;
+    }
+  }
+  .angular-ui-tree-placeholder {
+    border: 2px dashed #add8e6;
+    height: 70px;
+    margin-left: 50px;
+    width: 200px;
+  }
+  .drag-and-drop-sort-container {
+    border: 1px solid #000000;
+    margin: auto;
+    overflow: scroll;
+  }
+  .item {
```

Comment:
prefix this with `drag-and-drop` so it becomes `drag-and-drop-item` this is so that it's easy to tell that the element is associated with the Drag and Drop interaction

Reply:
Done